src/pgduckdb_ddl.cpp

```diff
@@ -371,16 +371,30 @@ DuckdbHandleDDL(PlannedStmt *pstmt, const char *query_string, ParamListInfo para
 		return;
 	} else if (IsA(parsetree, RenameStmt)) {
 		auto stmt = castNode(RenameStmt, parsetree);
-		if (stmt->renameType != OBJECT_SCHEMA) {
-			/* We only care about schema renames for now */
-			return;
-		}
-		if (strncmp("ddb$", stmt->subname, 4) == 0) {
-			elog(ERROR, "Changing the name of a ddb$ schema is currently not supported");
+		if (stmt->renameType == OBJECT_SCHEMA) {
+			if (strncmp("ddb$", stmt->subname, 4) == 0) {
```

Comment:
Can we add an `IsDuckDBSchema` in `string_utils`?

```c++
inline bool
IsDuckDBSchema(const char* s) {
  return strncmp("ddb$", s, 4) == 0;
}
```

```suggestion
			if (IsDuckDBSchema(stmt->subname)) {
```
(and replace all ~10 instances in our codebase?)

Reply:
Doing this one in a follow up PR